ee/clickhouse/models/util.py

```diff
@@ -58,12 +58,13 @@ def is_json(val):
     return True
 
 
-def is_int(val: str) -> bool:
+def is_int(value: Optional[Union[str, int]]) -> bool:
     try:
-        int(val)
-        return True
-    except ValueError:
+        int(value)
```

Comment:
Not sure what's the exact intention of this util func, but if it's really checking whether `value` is an integer, this should perhaps be `return value == int(value)`?